Curve-finance-suite/src/agents/curve.dao.killing.gauge.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from "forta-agent";
+
+import Web3 from "web3";
+import abi from "../utils/curve-gauge";
+
+// @ts-ignore
+import abiDecoder from "abi-decoder";
+abiDecoder.addABI(abi);
+
+export const web3 = new Web3();
+
+export const setKilled = {
+  name: "set_killed",
+  outputs: [],
+  inputs: [{ type: "bool", name: "_is_killed" }],
+  stateMutability: "nonpayable",
+  type: "function",
+  gas: 36878,
+};
+
+const createFinding = (alertId: string): Finding => {
+  return Finding.fromObject({
+    name: "Set Killed funciton called",
+    description: "Set Killed funciton called on pool",
```

Comment:
on what pool? Is the address field in metadata the pool? Can you add the pool in the metadata? Can you add any metadata to make this finding more informative?